AmpFinKit/Sources/AFNetwork/Methods/JellyfinClient+Playlist.swift

```diff
@@ -16,11 +16,12 @@ private let playlistQuery = [
 ]
 
 public extension JellyfinClient {
-    func create(playlistName: String, trackIds: [String]) async throws {
+    func create(playlistName: String, trackIds: [String], IsPublic: Bool) async throws {
```

Comment:
This is a minor thing, but swift argument labels should start in lowercase, not uppercase

Reply:
Resolved in a25ef92db13c33cb1c7ec1cd5f4957bb579f3266